src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -0,0 +1,66 @@
+"use client";
+import { Select as MuiSelect, SelectChangeEvent } from "@mui/material";
+import { MenuItem } from "@mui/material";
+import styled from "@emotion/styled";
+import type { SelectProps } from "@mui/material";
+
+export interface ISelect extends SelectProps {
+  items: Array<string>;
+  placeholder?: string;
+  currentSelectedData?: string[];
+  height?: number;
+  width?: number;
+  onChangeValue: (values: string[]) => void;
+}
+
+const MultiSelect = (props: ISelect) => {
+  const { items, placeholder = "None", height, width, currentSelectedData, onChangeValue, ...others } = props;
+
+  const handleChange = (event: SelectChangeEvent<unknown>) => {
+    const {
+      target: { value },
+    } = event;
+
+    const newValue: string[] = typeof value === "string" ? value.split(",") : (value as string[]);
+
+    onChangeValue(newValue);
+  };
+
+  return (
+    <>
+      <StyledSelect
+        multiple
+        value={currentSelectedData}
+        onChange={handleChange}
+        displayEmpty
+        renderValue={() => {
+          if (!currentSelectedData?.length) {
+            return <em>{placeholder}</em>;
+          }
+
+          return currentSelectedData.join(",");
+        }}
+        width={width}
+        height={height}
+        {...others}
+      >
+        {items.map((item: string, idx: number) => (
+          <MenuItem value={item} key={`${item}-${idx}`}>
+            {item}
+          </MenuItem>
```

Comment:
```suggestion
        {items.map((item, idx) => (
          <MenuItem value={item} key={`${item}-${idx}`}>
            {item}
          </MenuItem>
```

이건 따로 지정 안해줘도 타입이 잘 넘어갈 것 같아 !